package.json

```diff
@@ -144,6 +185,26 @@
         "category": "%commands.category.ansible-vault%",
         "command": "extension.ansible.vault",
         "title": "%commands.title.ansible-vault%"
+      },
+      {
+        "command": "ansible.lightspeed.inlineSuggest.accept",
+        "title": "Ansible Lightspeed: Ansible Lightspeed inline suggestion accept"
```

Comment:
Do we need Ansible Lightspeed twice? Should this just be:
Ansible Lightspeed: Inline suggestion accept